modules/check_in/spec/serializers/check_in/v2/appointment_identifiers_serializer_spec.rb

```diff
@@ -85,7 +86,8 @@
               patientDFN: '888',
               stationNo: '5625',
               icn: nil,
-              mobilePhone: '5553334444'
+              mobilePhone: '5553334444',
+              patientCellPhone: '4445556666'
```

Comment:
Do we have `patientCellPhone` being returned here?

Reply:
I set up the `appointment_identifiers_serializer` to return the patientCellPhone as an attribute under attributes. The patientCellPhone comes from CHIP under the `payload` object however